### PR TITLE
airshipper 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by printfn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This is the launcher/installer for [Veloren](https://gitlab.com/veloren/veloren). I recently tried to make a package for Veloren itself (#107328), but packaging the launcher instead might be a more feasible approach.

### CODE REVIEW

**Comment by printfn** [comment]
Hm, it looks like the [launcher's repository](https://gitlab.com/veloren/airshipper) isn't notable enough. The game's [main repo](https://gitlab.com/veloren/veloren) has over 1600 stars though.

File: Formula/airshipper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "client")
+    system "cargo", "install", *std_cargo_args(path: "server")
+  end
+
+  test do
+    assert_match "Done", shell_output("#{bin}/airshipper update")
```
Comment (carlocab) [inline Formula/airshipper.rb:16]: Does this software update itself? Or is it just updating some sort of local data store?

Comment (printfn) [reply Formula/airshipper.rb:16]: It doesn’t update itself. It merely downloads the executable and assets of Veloren.

**Comment by printfn** [comment]
I've now added the missing dependencies for Linux.

**Pull request closed without merging**